packages/js/package.json

```diff
@@ -36,15 +36,15 @@
   "scripts": {
     "clean": "rimraf dist",
     "build": "yarn clean && tsc && tsc-alias && tsc -p test/tsconfig.json && tsc-alias -p test/tsconfig.json && rollup -c",
+    "lint:test": "yarn eslint --ext js,ts,tsx src",
+		"lint:fix": "yarn eslint --fix --ext js,ts,tsx src",
     "test": "tape dist/test/**/*.test.js && yarn test:exports",
     "test:exports": "node ./test/cjs-export.test.cjs && node ./test/esm-export.test.mjs",
     "preversion": "yarn build",
     "postversion": "git push --follow-tags"
   },
   "dependencies": {
     "@bundlr-network/client": "^0.7.14",
-    "@metaplex-foundation/beet": "^0.4.0",
-    "@metaplex-foundation/beet-solana": "^0.3.0",
```

Comment:
Why are we removing these dependencies?

Reply:
It is not sued in the project. We use dep check to ensure that all deps are used and no deps are used without definition

Reply:
Fair. Although the PR I'm working on at the moment makes heavy use of them. Let's remove them for now and I'll add them back on my PR. Not that `beet` is being used in the tests so it at least needs to be a dev dependency.

Reply:
All dev deps placed in the root package json